configure.ac

```diff
@@ -1229,7 +1229,6 @@ AC_ARG_ENABLE([xmss],
     [ ENABLED_XMSS=no ]
     )
 
-ENABLED_WC_XMSS=no
```

Comment:
I think this `ENABLED_WC_XMSS=no` needs to be added back.

It's trying to build empty src files for wc_xmss.c and wc_xmss_impl.c even for just a `./configure && make` build.

Reply:
What empty source files?
wc_xmss.c and wc_xmss_impl.c now have implementations.

Reply:
I mean it's trying to build `wc_lms.c`, `wc_lms_impl.c`, `wc_xmss.c`, `wc_xmss_impl.c` even for just `./configure && make` when everything in those src files is gated out.

Reply:
Empty translation unit when no implementation is fixed.

Reply:
I think the empty translation unit error is happening because it's trying to build the wc_lms/wc_xmss src with everything gated out.

It should be the same pattern as wc_kyber configure I think:
```
AC_ARG_ENABLE([kyber],
    [AS_HELP_STRING([--enable-kyber],[Enable KYBER (requires --enable-experimental) (default: disabled)])],
    [ ENABLED_KYBER=$enableval ],
    [ ENABLED_KYBER=no ]
    )

ENABLED_WC_KYBER=no
for v in `echo $ENABLED_KYBER | tr "," " "`
do
  case $v in```

Reply:
Tests working now.
I moved the ifdef protection lower so that some more headers are always pulled in.